packages/components/src/MultiSelect/MultiSelect.module.css

```diff
@@ -48,8 +48,8 @@
 }
 
 .small {
-  --field--padding-left: calc(var(--space-base) - var(--space-smaller));
-  --field--padding-right: calc(var(--space-base) - var(--space-smaller));
+  --field--padding-left: var(--space-slim);
```

Comment:
These changes are identical to the FormField one and as a larger question, if MultiSelect is sticking around is there a way it could inherit/import styling from FormField rather than be re-declared?